twilight-gateway/README.md

```diff
@@ -138,7 +138,6 @@ There are a few additional examples located in the
 [codecov link]: https://app.codecov.io/gh/twilight-rs/twilight/
 [discord badge]: https://img.shields.io/discord/745809834183753828?color=%237289DA&label=discord%20server&logo=discord&style=for-the-badge
 [discord link]: https://discord.gg/7jj8n7D
-[docs:discord:sharding]: https://discord.com/developers/docs/topics/gateway#sharding
```

Comment:
Oops, my markdown linter auto removed this. I don't think it's the end of the world to sneak in too?